config/newrelic.yml

```diff
@@ -16,7 +20,9 @@
     explain_threshold: 0.5
   error_collector:
     enabled: true
-    capture_source: true
-    ignore_errors: ActionController::RoutingError
-  common: &default_settings
-    host: <%= Rails.application.secrets.newrelic[:endpoint] %>
+    ignore_classes: ['ActionController::RoutingError']
+  application_logging:
+    forwarding:
+      enabled: false
+  distributed_tracing:
+    enabled: false
```

Comment:
Is `distributed_tracing` being disabled due to associated costs? If so, can you add a note to https://cm-jira.usa.gov/browse/SRCH-2796? We were hoping to enable that at some point.

Reply:
`distributed_tracing` is disabled right now, but it's enabled by default starting with 8.x: https://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/migration-8x-guide/#distributed_tracing

I'm not sure what the associated cost is.